core/ui/mpc/session/StartMpcSessionStep.tsx

```
@@ -1,3 +1,5 @@
+import { isServer } from '@core/mpc/devices/localPartyId'
+import { KeygenType } from '@core/mpc/keygen/KeygenType'
```

ESLint

[error] 'KeygenType' is defined but never used.

(@typescript-eslint/no-unused-vars)